public/index.html

```diff
@@ -1,14 +1,21 @@
 <!DOCTYPE html>
-<html lang="fi">
+<html prefix="og: http://ogp.me/ns#" lang="fi">
```

Comment:
Where is that link heading to? Can we host that xml file ourselve?

Reply:
This is taken from the Open Graph protocol site: https://ogp.me/